### PR TITLE
update(bump-regex): v4.1

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- `keepmetadata` option
- version bump

https://github.com/stevelacy/bump-regex/compare/2.9.0...4.0.0

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by silkentrance — COMMENTED** [review]
The latest version is 4.1.0.
Can you please update the version information before merging this?
BTW hell, this is years ago, never used bump regex since then xD

File: types/bump-regex/index.d.ts [hunk 1]
```diff
@@ -1,18 +1,23 @@
-// Type definitions for bump-regex 2.9
+// Type definitions for bump-regex 4.0
```
Comment (peterblazejewicz) [inline types/bump-regex/index.d.ts:1]:
```suggestion
// Type definitions for bump-regex 4.1
```

**Comment by peterblazejewicz** [comment]
> Can you please update the version information before merging this?

Done, thx!

**Review by silkentrance — APPROVED** [review]

**Comment by peterblazejewicz** [comment]
Ready to merge 💘

**Pull request merged**